### ISSUE TITLE
Some a11y strings are untranslatable

### ISSUE THREAD

**Issue by phet-steele** [phetsims/friction#57]
These three strings look to not be translatable: "Move Book", "Move book", and "Move in smaller steps".

<img width="848" alt="screen shot 2017-12-07 at 2 31 04 pm" src="https://user-images.githubusercontent.com/5863899/33740484-16ee7bb6-db5e-11e7-849a-ac4dc0514041.png">

Seen on macOS 10.13.1. For phetsims/QA/issues/75.

<details>
<summary>Troubleshooting Information</summary>

URL: https://www.colorado.edu/physics/phet/dev/html/friction/1.4.0-rc.2/friction_en.html?stringTest=%20
Version: 1.4.0-rc.2 2017-12-05 18:19:27 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.84 Safari/537.36
Language: en-US
Window: 1920x1013
Pixel Ratio: 2/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 32 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 80)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {"assert":{"sha":"928741cf","branch":"HEAD"},"axon":{"sha":"d70f5056","branch":"HEAD"},"babel":{"sha":"26366701","branch":"master"},"brand":{"sha":"53cf227b","branch":"HEAD"},"chipper":{"sha":"6f8b4c13","branch":"HEAD"},"dot":{"sha":"f09e44aa","branch":"HEAD"},"friction":{"sha":"d2fa6e36","branch":"HEAD"},"joist":{"sha":"bc8f433b","branch":"HEAD"},"kite":{"sha":"72dcc8ee","branch":"HEAD"},"phet-core":{"sha":"88a51822","branch":"HEAD"},"phetcommon":{"sha":"f0288123","branch":"HEAD"},"query-string-machine":{"sha":"b8bdcc28","branch":"HEAD"},"scenery":{"sha":"31c7d26f","branch":"HEAD"},"scenery-phet":{"sha":"594695db","branch":"HEAD"},"sherpa":{"sha":"0f5f30f7","branch":"HEAD"},"sun":{"sha":"2d4b1ff7","branch":"HEAD"},"tandem":{"sha":"56c77424","branch":"HEAD"}}
</details>

**Comment by jessegreenberg** [comment]
@zepumph would you mind fixing this?

**Comment by zepumph** [comment]
@jessegreenberg it was agreed that we don't care about the wasd keys? What if another language has different keys for those positions?

**Comment by jessegreenberg** [comment]
Ah, right, Good point @zepumph. But I think the report still applies for "Move Book", "Move book", and "Move in smaller steps" labels on the left side.

**Comment by zepumph** [comment]
Sorry I'm not being clear. I agree about the three on the side. 

I'm wondering about the wasd keys. They aren't currently, but should they be?

**Comment by jessegreenberg** [comment]
No, WASD keys  should not be translatable because we currently do not have a way to map different keycodes with the translation utility.

**Comment by zepumph** [comment]
Is a keyCode tied to the letter, or to the place on the keyboard? http://mentalfloss.com/article/52483/6-non-qwerty-keyboard-layouts

I'm wondering if we need to make those translatable to support different looking, international keyboard layouts. 

Even so, because of your concern, we would just have to hope that our translators punch in the right keys (the keys needed to move the book, not the literal translation for a 'W').

**Comment by zepumph** [comment]
The three in the panel should be fixed now
![image](https://user-images.githubusercontent.com/6856943/33747769-5c6d07f8-db71-11e7-8a05-4e113888ae36.png)

**Comment by zepumph** [comment]
@jessegreenberg please unassign all and mark "fixed pending testing" if you don't think we should work on https://github.com/phetsims/friction/issues/57#issuecomment-350138352 right now.

**Comment by jessegreenberg** [comment]
I would assume it is the code for the letter, not the layout on the keyboard, but I don't know for sure. For instance on a Colemak layout, I doubt that the code for R becomes the code for S.

I am not sure yet how we will support layouts like JCUKEN (russian keyboard in your link), keycodes could be very different. I think we should prevent i18n of keys until we decide to support different keycodes in the translation utility.

Hopefully all arrow keys are the same.

**Comment by phet-steele** [comment]
Thanks @zepumph!
